util/Migrator/DbScripts/2021-06-15_00_UseResetPasswordCustomOrg.sql

```diff
@@ -0,0 +1,6 @@
+-- For Enterprise (annual/monthly) and Custom (internal) orgs, enable potential use of UseResetPassword
+UPDATE
+    [dbo].[Organization]
+SET
+    [UseResetPassword] = (CASE WHEN [PlanType] = 10 OR [PlanType] = 11 OR [PlanType] = 6 THEN 1 ELSE 0 END)
```

Comment:
So, we don't want, retroactively, if this is enabled off-cycle for a different org type by Customer Success, for a deployment to overwrite the value already in the field (so basically `1` OR `{existing value}`).
Also, you can use the `IN ()` keyword to increase readability.
I would suggest something like the following:
```sql
[UseResetPassword] = (CASE WHEN [PlanType] IN (10, 11, 6) THEN 1 ELSE [UseResetPassword] END)
```

Reply:
Thank you! Will update.